images/installer-origin-release/Dockerfile

```diff
@@ -0,0 +1,17 @@
+# This Dockerfile is used to create the image quay.io/coreos/origin-release:bazel used in CI
+# to build smoke tests.
+
+FROM docker.io/openshift/origin-release:golang-1.10
+
+COPY bazel-copr-vbatts.repo /etc/yum.repos.d/
```

Comment:
Do we want to copy this from version control?  I'd rather `curl` it directly from [upstream][1].

[1]: https://copr.fedorainfracloud.org/coprs/vbatts/bazel/repo/epel-7/vbatts-bazel-epel-7.repo